src/class_singleWorker.py

```diff
@@ -1120,7 +1139,14 @@ def sendMsg(self):
                 privSigningKeyHex, privEncryptionKeyHex, \
                     pubSigningKey, pubEncryptionKey = self._getKeysForAddress(
                         fromaddress)
-            except:
+            except (configparser.NoSectionError, configparser.NoOptionError) as err:
+                self.logger.warning("Section or Option did not found: %s", err)
+            except Exception as err:
+                self.logger.error(
+                    'Error within sendMsg. Could not read'
+                    ' the keys from the keys.dat file for a requested'
+                    ' address. %s\n', err
+                )
                 queues.UISignalQueue.put((
```

Comment:
this section I would put into the previous except

Reply:
and in this except, put something like `error, can't send` into the `UISignalQueue`. In the log file, maybe put whole error info.

Reply:
also in `sendBroadcast`

Reply:
Updated